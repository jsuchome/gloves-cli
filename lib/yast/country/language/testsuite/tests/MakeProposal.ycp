/**
 * tests for Language::MakeProposal
 * Author:	jsuchome@suse.cz
 * $Id: MakeProposal.ycp 32769 2006-09-08 09:29:15Z jsuchome $
 */
{

    // testedfiles: Language.ycp

    include "testsuite.ycp";

    map READ = $[
	"sysconfig" : $[
	    "language" : $[
		"RC_LANG"		: "en_US.UTF-8",
		"INSTALLED_LANGUAGES"	: "en_US,de_DE",
	    ],
	],
	"target" : $[
	    "bash_output"	: $[],
	    "size"		: 1,
	    "dir" : [
		"language_de_DE.ycp"
	    ],
	    "yast2" : $[
		"de_DE" : [
		    "Deutsch",
		    "Deutsch",
		    ".UTF-8",
		    "@euro",
		    "German"
		],
	    ],
	],
    ];

    TESTSUITE_INIT([READ,$[],READ], nil);

    import "Language";

    TEST(``(Language::MakeProposal (false, false)), [READ,$[],$[]], []);

    Language::languages	= "en_US";

    TEST(``(Language::MakeProposal (true, false)), [READ,$[],$[]], []);
}
