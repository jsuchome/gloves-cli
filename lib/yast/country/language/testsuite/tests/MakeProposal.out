Read	.target.size "/etc/sysconfig/language" 1
Read	.target.dir "/usr/share/YaST2/data/languages" ["language_de_DE.ycp"]
Read	.target.yast2 "languages/language_de_DE.ycp" $["de_DE":["Deutsch", "Deutsch", ".UTF-8", "@euro", "German"]]
Return	["Primary Language: English (US)", "Additional Languages: German"]
Return	["Primary Language: English (US)"]
